### ISSUE TITLE
Overlay frame transitions do not maintain relative scale - continued

### ISSUE THREAD

**Issue by kamunimaximus** [simonoliver/UnityFigmaBridge#12]
I am unable to close an issue which has been closed by someone else so I am referencing the issue here. Feel free to delete this thread an re-open the last one. Sorry for any inconvenience. 

 > @kamunimaximus Curious about your use case here - right now I've assumed that all screens would be the same size (as you are designing for the same device). It looks like you want to have a modal popup appear on one of your screens. For this I'd recommend having this as a component that is placed on an existing screen, and simply made made visible in code. Let me know if you want an example for this.

Sorry for the late reply! The use case is for sure a modal, when setting a frame as overlay in figma you can specify overlay background and behaviour for the modal component. Right now the behaviour in figma and unity is vastly different

Let me know if this makes sense

_Originally posted by @kamunimaximus in https://github.com/simonoliver/UnityFigmaBridge/issues/4#issuecomment-1479256228_

**Comment by simonoliver** [comment]
Yep, makes perfect sense, thanks for the clarification - right the package only implements "Navigate to" interactions. I'm going to change this to an enhancement to  add support for other interaction types.

**Comment by simonoliver** [comment]
Currently only "Navigate to" is supported

We need to implement:

- Change to
- Open Overlay
- Swap Overlay
- Close Overlay
- Back
- Scroll to
- Open Link

I believe right now this isn't exposed in the Figma API but will investigate
